config-linux.md

```diff
@@ -384,6 +384,30 @@ The following parameters can be specified to set up the controller:
     }
 ```
 
+### <a name="configLinuxVTPMs" />vTPMs
+
+**`vtpms`** (array of objects, OPTIONAL) lists a number of emulated TPMs that will be made available to the container.
+
+Each entry has the following structure:
+
+* **`statePath`** *(string, REQUIRED)* - a directory for persisting vTPM state. This value MUST be an absolute path.
+* **`vtpmVersion`** *(string, OPTIONAL)* - The version of TPM to emulate, either 1.2 or 2; default is 1.2.
+* **`createCerts`** *(boolean, OPTIONAL)* - If true then create certificates for the vTPM, defaults to false.
+
+The `statePath` MUST be unique per container. If the `vtpms` array contains duplicate entries with the same `statePath`, the runtime MUST generate an error.
```

Comment:
These two sentences [should be on two lines][1].

And I think the two sentences you have for the `statePath` list entry above should be as well:

```
* **`statePath`** *(string, REQUIRED)* - a directory for persisting vTPM state.
    This value MUST be an absolute path.
```

(with a four-space indent, #495, #846), although the current spec is not always consistent about list entries (e.g. [here we have two sentences in one list-entry line][2]).

Also, existing examples use a link for “generate an error” (e.g. see [here][3]).

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0/style.md#one-sentence-per-line
[2]: https://github.com/opencontainers/runtime-spec/blame/v1.0.0/config-linux.md#L272
[3]: https://github.com/opencontainers/runtime-spec/blame/v1.0.0/config-linux.md#L452

Reply:
Thanks for your patience. Fixed.